qiskit/pulse/schedule.py

```diff
@@ -716,6 +717,21 @@ def assign_parameters(self,
 
         return self
 
+    def get_parameter(self,
+                      parameter_name: str) -> List[Parameter]:
```

Comment:
```suggestion
    def get_parameters(self,
                       parameter_name: str) -> List[Parameter]:
```
I think this should be plural, since it returns a list.